Fixing glossary and index in spanish (#364)
gvwilson/teachtogether.tech: es/pck.tex

  viven en sociedades occidentales, democráticas, industrializadas y con alto nivel de riqueza y educación] y se los denomina WEIRD\index{WEIRD} por \emph{Western, Education, Industrialized, Rich, and Democratic en inglés}, tal como señala ~\cite{Henr2010}.
  Además,
  son principalmente jóvenes que estudian en instituciones educativas, ya que es la población a la que la mayoría de las personas que investigan tienen fácil acceso.

>   Sabemos mucho menos sobre adultos, grupos marginados y estudiantes en ambientes educativos flexibles (estudiantes free-range), así como sobre \grefdex{g:end-user-programmer}{usuarias/os finales programadoras/es}{usuario/a final programador/a},

Reviewer: masc/fem

Author: ¿Cómo pongo programador/a en singular con femenino antes que masculino? ¿programadora/or ?

>   Sabemos mucho menos sobre adultos, grupos marginados y estudiantes en ambientes educativos flexibles (estudiantes free-range), así como sobre \grefdex{g:end-user-programmer}{usuarias/os finales programadoras/es}{usuario/a final programador/a},

Author: ```suggestion
  Sabemos mucho menos sobre adultos, grupos marginados y estudiantes en ambientes educativos flexibles (estudiantes free-range), así como sobre \grefdex{g:end-user-programmer}{usuarias/os finales programadoras/es}{usuario/a final programadora/or},
```

  hacer que las personas novatas trabajen en ejercicios de creatividad computacional mejora sus calificaciones en varios niveles~\cite{Shel2017}.
  Un ejercicio típico es describir un objeto cotidiano (como un clip o un cepillo de dientes)
  en términos de sus entradas, salidas y funciones.

>   Este tipo de enseñanza a veces se es llamada \gref{g:cs-unplugged}{CS desconectado};

Author: ```suggestion
  Este tipo de enseñanza a veces se es llamada \grefdex{g:cs-unplugged}{CS desconectado}{ciencias de la computación desconectado};
```

>   Este tipo de enseñanza a veces se es llamada \grefdex{g:cs-unplugged}{CS desconectado}{ciencias de la computación desconectado};

Reviewer: ```suggestion
  Este tipo de enseñanza a veces es llamada \grefdex{g:cs-unplugged}{CS desconectado}{ciencias de la computación desconectada};
```

Reviewer: quizás es mejor "ciencias de la computación sin conexión"?

Author: o desenchufada....que sería la traducción más literal??....me gusta sin conexión también
